example/linux/installation.md

```
@@ -0,0 +1,66 @@
+1. **Create the Systemd Service File**:
+   - Create a file named `talawa-api.service` in the `/etc/systemd/system` directory. Ensure you have root privileges.
+
+2. **Edit the `talawa-api.service` File**:
+   - Modify the `talawa-api.service` file as required (e.g., `ExecStart`, `WorkingDirectory`, `User`, `Group`). You can find how to edit these fields in the `talawa-api.service` file located in `example/linux/systemd`.
+
+3. **Copy the Configuration**:
+   - Copy the text into the `talawa-api.service` file located in `/etc/systemd/system` and save it.
+
+4. **Edit the `Talawa-api.sh` Script**:
+   - Edit the `Talawa-api.sh` script to set the project directory and log file path:
+   
```

Markdownlint (0.37.0)

Expected: 0 or 2; Actual: 3
Trailing spaces

(MD009, no-trailing-spaces)